tmd/Topology/methods.py

```diff
@@ -169,13 +169,11 @@ def extract_ph_neuron(
 
     if sort:
         sort_ph(ph)
-    else:
-        p = ph
```

Comment:
This is weird. What should happen if sort is true?? @lidakanari

Reply:
Good point :thinking:

Reply:
yep looks good now.